app/views/documents/show.html.erb

```diff
@@ -2,22 +2,25 @@
 <% content_for :title, @edition.title_or_fallback %>
 <% content_for :context, @edition.document_type.label %>
 
-<%= render "govuk_publishing_components/components/tabs", {
-  panel_border: false,
-  tabs: [
+<%= render "components/secondary_navigation", {
+  aria_label: "Document",
+  items: [
     {
-      id: "edit-document",
       label: t("documents.show.tabs.summary"),
-      title: t("documents.show.tabs.summary"),
-      content: render("documents/show/summary_tab"),
-      tab_data_attributes: { gtm: "summary-tab" }
+      href: document_path,
+      current: true,
+      data_attributes: {
+        gtm: "summary-tab"
+      }
     },
     {
-      id: "document-history",
       label: t("documents.show.tabs.history"),
-      title: t("documents.show.tabs.history"),
-      content: render("documents/show/history_tab"),
-      tab_data_attributes: { gtm: "history-tab" }
+      href: history_path_url,
+      data_attributes: {
+        gtm: "history-tab"
+      }
     }
   ]
 } %>
```

Comment:
I expect this component will end up in a partial where we specify a parameter of which one is current.